posts/2024-03-04_inside_the__phar.../inside_the__pharmaverse.qmd

```diff
@@ -0,0 +1,136 @@
+---
+title: "Inside the Pharmaverse"
+author:
+  - name: Michael Rimler
+description: "A short blog to help the Pharmaverse community understand how Pharmaverse is governed"
+date: "2024-03-04"
+# please do not use any non-default categories.
+# You can find the default categories in the repository README.md
+categories: [community]
+# feel free to change the image
+image: "pharmaverse.png"
+---
+
+<!--------------- typical setup ----------------->
+
+```{r setup, include=FALSE}
+long_slug <- "2024-03-04_inside_the__phar..."
+# renv::use(lockfile = "renv.lock")
+```
+
+<!--------------- post begins here ----------------->
+
+Greetings, Pharmaverse phriends!
+
+Now that the {pharmaverse} has been around for a few years and seems to be making an impact on clinical reporting (yay!), we wanted to take an opportunity to cast a light on some of the inner workings of Pharmaverse and its Council (us).
+Fortunately, some of you (you know who you are) innovated and created this fantastic channel for [Pharmaverse Blogs](https://pharmaverse.github.io/blog/).
+We can’t think of a better way to communicate than by the channel that the Pharmaverse community itself created.
+
+## Pharmaverse Inception
+
+The Pharmaverse Council was formed in 2020 as a coming together of 4 like-minded individuals embedded in the industry-wide efforts to increase the use of the R language for clinical trial analysis and reporting.
+These individuals included Ross Farrugia (Roche), Sumesh Kalappurakal (Johnson & Johnson), Michael Rimler (GSK), and Mike Stackhouse (Atorus).
+Each had historical experience with R adoption, through their primary roles within their respective organizations.
```

Comment:
Pooja's ISCR post cross-links to linkedin - maybe can do some here for those willing?  I think folks reading this post would be interested in following you four!

Reply:
I'll add in those that I have.